### DISCUSSION TITLE
Dependabot: Disable by default on repo's generated from a template repo

### DISCUSSION THREAD

**Discussion by jeroenverfallie** [community/community#38562] [Code Security]
Is there any chance dependabot will be disabled by default on repositories generated from a template repo in the near future, now that we have this [feature on forks](https://github.blog/changelog/2022-11-07-dependabot-pull-requests-off-by-default-for-forks/)?

**Comment by ossanna16** [comment]
Hi there @jeroenverfallie and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
